.github/workflows/V8-reanimated-build-check-nightly.yml

```diff
@@ -29,13 +29,13 @@ jobs:
           distribution: 'zulu'
           java-version: '17'
       - name: Create React Native app
-        run: npx react-native init app
```

Comment:
`npx react-native init`/`rncli init` can be unstable at times. Installing it firstly with yarn, then doing `yarn rncli init` improves stability for some reasons. Please adjust to that way of app initialization here.